### PR TITLE
explicitly inline, silence warning

### PR DESCRIPTION
**Pull request by VinInn**

### CODE REVIEW

**Comment by VinInn** [comment]
@cmsbuild , please test

File: RecoTracker/FinalTrackSelectors/plugins/getBestVertex.h [hunk 1]
```diff
@@ -2,11 +2,9 @@
 #include "DataFormats/VertexReco/interface/Vertex.h"
 #include <limits>
 
-namespace {
```
Comment (slava77) [inline RecoTracker/FinalTrackSelectors/plugins/getBestVertex.h]: is this removal really harmless?
Would there be nothing else to include this file and have a possibly different meaning of Point?

Comment (VinInn) [inline RecoTracker/FinalTrackSelectors/plugins/getBestVertex.h]: anonymous namespace is prohibited in header files...
strictly speaking also the using, still the reality is that this file is included in only two files in this very plugin directory

Comment (slava77) [inline RecoTracker/FinalTrackSelectors/plugins/getBestVertex.h]: @Dr15Jones 
this file has "using X = Y;"
static analyzer does not report it as an issue.
Should it be?

**Comment by slava77** [comment]
+1

for https://github.com/cms-sw/cmssw/pull/16674 42fe17c
- technical update
- no changes in jenkins tests

**Comment by davidlange6** [comment]
+1

**Pull request merged**